lib/pages/emergency_dashboard.dart

```diff
@@ -68,6 +70,24 @@ class _HomState extends State<Hom> with SingleTickerProviderStateMixin {
             child: Column(
               // mainAxisAlignment = MainAxisAlignment.spaceEvenly
               children: [
+                Container(
+                  alignment: Alignment.centerRight,
+                  padding: EdgeInsets.only(top: 8.0),
+                  child: IconButton(
+                    icon: Icon(Icons.exit_to_app_sharp, color: Colors.red, size: 30.0,),
+                    onPressed: ()async{
+                      var _gAuth = GoogleAuthenticate(context);
+                      bool response = await _gAuth.logOut();
+                      if(response){
+                        Navigator.pushAndRemoveUntil(
```

Comment:
Please use the routeing mechanism to navigate between screens. Have a look in another place in project how it is done.